aws/src/main/java/org/apache/iceberg/aws/s3/S3FileIOProperties.java

```diff
@@ -684,6 +715,22 @@ private Set<Tag> toS3Tags(Map<String, String> properties, String prefix) {
         .collect(Collectors.toSet());
   }
 
+  public boolean isS3AccessGrantsEnabled() {
+    return isS3AccessGrantsEnabled;
+  }
+
+  public void setS3AccessGrantsEnabled(boolean s3AccessGrantsEnabled) {
+    this.isS3AccessGrantsEnabled = s3AccessGrantsEnabled;
+  }
+
+  public boolean isS3AccessGrantsFallbackToIAMEnabled() {
+    return isS3AccessGrantsFallbackToIAMEnabled;
+  }
+
+  public void setS3AccessGrantsFallbackToIAMEnabled(boolean s3AccessGrantsFallbackToIAMEnabled) {
```

Comment:
nit: `Iam` instead of `IAM` to be consistent with other codebase

Reply:
similar comment for the variable name

Reply:
sure, done.